controllers/volumereplicationgroup_controller.go

```diff
@@ -51,6 +51,10 @@ type VolumeReplicationGroupReconciler struct {
 	Scheme *runtime.Scheme
 }
 
+const (
+	vrgFinalizerName = "volumereplicationgroup.storage.io"
```

Comment:
Should we name this `volumereplicationgroups.ramendr.openshift.io/vrg-protection`? Going along the lines of <API-group>/<name>

Reply:
I see that my comment was cut off now... this is fine, I was looking at the name as `volumereplicationgroups.ramendr.openshift.io/vrg-protection` we can change it subsequently.